supabase/tests/65_test_github_user_id.sql

```
@@ -37,6 +62,101 @@ SELECT is(
+    WHERE conrelid = 'public.users'::regclass
+      AND conname = 'users_github_id_key'
+  ),
+  'u',
+  'users.github_id has the named unique constraint'
+);
+
+SELECT ok(
+  EXISTS (
+    SELECT 1
+    FROM pg_catalog.pg_constraint AS unique_constraint
+    JOIN pg_catalog.pg_index AS index_meta
```

SQLFluff (4.2.2)

[error] Join clauses should be fully qualified.

(AM05)